cmd/exo/state.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/deref/exo/internal/core/api"
+	"github.com/deref/exo/internal/util/jsonutil"
+	"github.com/deref/exo/internal/util/which"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(stateCmd)
+	stateCmd.AddCommand(stateGetCmd)
+	stateCmd.AddCommand(stateSetCmd)
+	stateCmd.AddCommand(stateClearCmd)
+	stateCmd.AddCommand(stateEditCmd)
+
+	stateCmd.AddCommand(helpSubcmd)
+}
+
+var stateCmd = &cobra.Command{
+	Use:    "state",
+	Short:  "View and update the state store.",
+	Long:   `Contains subcommands for getting, setting, and clearing state on a per-component basis.`,
+	Hidden: true,
+	Args:   cobra.MaximumNArgs(0),
```

Comment:
```suggestion
	Args:   cobra.NoArgs,
```
I think this returns a slightly clearer error.